src/pages/home/ReportScreen.js

```diff
@@ -285,105 +305,92 @@ function ReportScreen({
+                            <Banner
+                                containerStyles={[styles.mh4, styles.mt4, styles.p4, styles.bgDark]}
+                                textStyles={[styles.colorReversed]}
+                                text={translate('reportActionsView.chatWithAccountManager')}
+                                onClose={dismissBanner}
+                                onPress={chatWithAccountManager}
+                                shouldShowCloseButton
+                            />
+                        )}
+                        <DragAndDropProvider isDisabled={!isReportReadyForDisplay}>
+                            <View
+                                style={[styles.flex1, styles.justifyContentEnd, styles.overflowHidden]}
+                                onLayout={calculateSkeletonViewHeight}
+                            >
+                                {isReportReadyForDisplay && !isLoadingInitialReportActions && !isLoading && (
+                                    <ReportActionsView
                                         reportActions={reportActions}
                                         report={report}
+                                        isLoadingReportActions={reportMetadata.isLoadingReportActions}
+                                        isLoadingMoreReportActions={reportMetadata.isLoadingMoreReportActions}
                                         isComposerFullSize={isComposerFullSize}
-                                        onSubmitComment={onSubmitComment}
-                                        policies={policies}
+                                        parentViewHeight={skeletonViewContainerHeight}
+                                        policy={policy}
                                     />
-                                </>
-                            )}
+                                )}
 
-                            {!isReportReadyForDisplay && (
-                                <ReportFooter
-                                    shouldDisableCompose
-                                    isOffline={network.isOffline}
-                                />
-                            )}
-                        </View>
-                    </DragAndDropProvider>
-                </FullPageNotFoundView>
-            </ScreenWrapper>
-        </ReportScreenContext.Provider>
+                                {/* Note: The ReportActionsSkeletonView should be allowed to mount even if the initial report actions are not loaded. If we prevent rendering the report while they are loading then
+                            we'll unnecessarily unmount the ReportActionsView which will clear the new marker lines initial state. */}
+                                {(!isReportReadyForDisplay || isLoadingInitialReportActions || isLoading) && <ReportActionsSkeletonView containerHeight={skeletonViewContainerHeight} />}
```

Comment:
I think nothing is shown not even the header because of onyx blocking the rendering, not because this conditional changed, but I can double check

Reply:
Yep, the conditional is correct on the first render of the screen, it just seems that now the actions load so fast that the skeleton is no longer rendered. But we are still working on finding more performance improvement because the screen can be mounted even faster, then the skeleton will probably start working again